contracts/mirror_mint/src/positions.rs

```diff
@@ -580,10 +579,8 @@ pub fn burn<S: Storage, A: Api, Q: Querier>(
     // If the collateral is default denom asset and the asset is deprecated,
     // anyone can execute burn the asset to any position without permission
     let mut close_position: bool = false;
-    if asset_config.end_price.is_some() {
-        let oracle: HumanAddr = deps.api.human_address(&config.oracle)?;
-        let asset_price: Decimal =
-            load_asset_price(&deps, &oracle, &position.asset.info, Some(env.block.time))?;
```

Comment:
We don't need it anymore because now we query `asset_price` and `collateral_price` separately.